docs/modules/ROOT/pages/utilities.adoc

```diff
@@ -188,3 +188,49 @@ await instance.multicall([
     instance.contract.methods.bar().encodeABI()
 ]);
 ----
+
+=== StaticDelegateCall
+
+The `StaticDelegateCall` abstract contract provides emulation of static delegatecalls for implementing view-only functions that execute another contract's code with the calling contract's context. It can be useful in implementing dynamic library support at runtime.
+
+Example:
+[source,solidity]
+----
+// contracts/DynamicGadget.sol
+// SPDX-License-Identifier: MIT
+
+interface IGadget {
+    function compute(uint256 c) external view returns (uint256);
+}
+
+contract GadgetStorage {
+    uint256 internal _x;
+}
+
+contract GadgetImplementationA is GadgetStorage, IGadget {
+    function compute(uint256 c) external view returns (uint256) {
+        return _x * c;
+    }
+}
+
+contract GadgetImplementationB is GadgetStorage, IGadget {
+    function compute(uint256 c) external view returns (uint256) {
+        return _x / c;
+    }
+}
```

Comment:
Why do we need two?

In case we don't need them, wouldn't be a clearer example something like:?

```solidity
contract GadgetImplementation {
   uint256 internal _x;

    function compute(uint256 c) external view returns (uint256) {
        return _x * c;
    }
}
```

Reply:
Yeah, I can simplify it to one if it gets the point across.

Reply:
Addressed in 007f271d.